enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_manager.py

```diff
@@ -43,6 +43,11 @@ def __init__(self, config):
 
         super().__init__(config)
         self.proof_data_type = config.get("WorkerConfig")["ProofDataType"]
+        workloads_str = config.get("WorkerConfig")["workloads"]
+        workloads = workloads_str.split(',')
+        self._workloads = []
+        for workload in workloads:
+            self._workloads.append(workload.encode("UTF-8").hex())
```

Comment:
Can we move this base enclave manager class. This is common for Singleton and WPE